rust/protocol/src/consts.rs

```diff
@@ -1,10 +1,56 @@
 //
-// Copyright 2020 Signal Messenger, LLC.
+// Copyright 2020-2021 Signal Messenger, LLC.
```

Comment:
I'm not sure if someone from outside Signal Messenger, LLC should be changing this, but yes it should be updated.

Reply:
I think because of the [CLA](https://signal.org/cla/) it will become correct upon merge, but good thought.